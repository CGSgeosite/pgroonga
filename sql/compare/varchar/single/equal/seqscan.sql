CREATE TABLE memos (
  id integer,
  title varchar(1023)
);

INSERT INTO memos VALUES (1, 'PostgreSQL');
INSERT INTO memos VALUES (2, 'Groonga');
INSERT INTO memos VALUES (3, 'PGroonga');

CREATE INDEX grnindex ON memos USING pgroonga (title pgroonga.varchar_ops);

SET enable_seqscan = on;
SET enable_indexscan = off;
SET enable_bitmapscan = off;

SELECT id, title
  FROM memos
 WHERE title = 'Groonga';

DROP TABLE memos;
