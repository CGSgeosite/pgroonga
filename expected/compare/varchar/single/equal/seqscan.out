CREATE TABLE memos (
  id integer,
  title text
);
INSERT INTO memos VALUES (1, 'PostgreSQL');
INSERT INTO memos VALUES (2, 'Groonga');
INSERT INTO memos VALUES (3, 'PGroonga');
CREATE INDEX grnindex ON memos USING pgroonga (title pgroonga.varchar_ops);
SET enable_seqscan = on;
SET enable_indexscan = off;
SET enable_bitmapscan = off;
SELECT id, title
  FROM memos
 WHERE title = 'Groonga';
 id |  title  
----+---------
  2 | Groonga
(1 row)

DROP TABLE memos;
